recipe/meta.yaml

```diff
@@ -1,71 +1,65 @@
+{% set version = "1.1.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 3e682e8ea6e1ee26a92cf289f207d539f30e44879126c128ff8f4e360eb25a8b
+  sha256: f8d6d45a4391fdf740c7b74f79a72313f95a5f6e3417eaf1c68892b47b920646
 
 build:
   number: 0
-  # As of 3/4/2022, we don't have the deps built out to support a py310 build.
-  skip: true  # [py==310]
-  script: "{{ PYTHON }} -m pip install . -vv"
+  # LunarCalendar, which is a noarch package, requires ephem, unavailable on s390x
+  # https://blog.quantinsti.com/installing-prophet-library-windows/
+  skip: true  # [py<39 or (linux and s390x)]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   binary_has_prefix_files:
-    - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.pkl
+    - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.bin
+  # To use our own cmdstan rather than download and build a different one:
+  script_env:
+    - PROPHET_REPACKAGE_CMDSTAN=false
+    - BOOST="${LIBRARY_INC}"  # [win]
 
 requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - m2w64-make                   # [win]
+    - m2-coreutils                 # [win]
   host:
     - python
+    - cmdstanpy 1.1.0
+    - numpy {{ numpy }}
     - pip
-    # Explanation:
-    #
-    # setup.py imports fbprophet.models.StanBackendEnum, triggering
-    # fbprophet/__init__.py and thus making the runtime dependencies
-    # also build-time dependencies.
-    #
-    # These requirements are copied from python/requirements.txt
-    - cython >=0.22
-    - pystan ~=2.19.1.1
-    - numpy >=1.15.4
-    - pandas >=1.0.4
-    - matplotlib-base >=2.0.0
-    - LunarCalendar >=0.0.9
-    - convertdate >=2.1.2
-    - holidays >=0.10.2
-    - setuptools-git >=1.2
-    - python-dateutil >=2.8.0
-    - tqdm >=4.36.1
+    - setuptools
+    - wheel
+    - tbb-devel 2021.8.0
+    - cmdstan 2.31.0
+    - libboost 1.73
   run:
     - python
-    - cython >=0.22
-    - pystan ~=2.19.1.1
-    - pandas >=1.0.4
-    - matplotlib-base >=2.0.0
-    - LunarCalendar >=0.0.9
+    - {{ pin_compatible('numpy') }}
+    - cmdstanpy >=1.0.4
     - convertdate >=2.1.2
-    - holidays >=0.10.2
-    - setuptools-git >=1.2
+    - holidays >=0.14.2
+    - LunarCalendar >=0.0.9
+    - matplotlib-base >=2.0.0
+    - pandas >=1.0.4
     - python-dateutil >=2.8.0
     - tqdm >=4.36.1
-    - {{ pin_compatible('numpy') }}
+    - importlib_resources
+    - cmdstan
 
 test:
-  imports:
-    - prophet
-    - prophet.tests
-  commands:
-    # 3/4/2022: Turning off 'pip check' because the requirements.txt
-    # lists 'cmdstanpy' which is an experimental backend that we are
-    # not supporting at this time.
-    #- pip check
-    - py.test -v --pyargs prophet
   requires:
     - pip
     - pytest
+    - notebook
+    - plotly
+  commands:
```

Comment:
Move `pip check` from commands to `run_test.py` because of the linter warning:
```
import subprocess

subprocess.run(["pip", "check"])
```

Reply:
done